src/content/release/breaking-changes/uiscenedelegate.md

```diff
@@ -0,0 +1,187 @@
+---
+title: UISceneDelegate adoption
+description: >
+  A guide for Flutter iOS developers to adopt Apple's UISceneDelegate protocol.
+---
+
+## Summary
+
+Apple is requiring iOS developers to adopt the UISceneDelegate protocol which
+changes the order of initialization when applications launch. Certain usages of
+Flutter inside `application:didFinishLaunchingWithOptions:` on iOS will have to
+be updated.
+
+## Background
+
+Most Flutter users will not have custom logic inside of
+`application:didFinishLaunchingWithOptions:`. Those apps will not need to do
+any migration.
+
+Apple is requiring the adoption of the `UISceneDelegate` which reorders the
+initialization of iOS apps. After a `UISceneDelegate` is specified,
+initialization of the Storyboard will be delayed until after
+`application:didFinishLaunchingWithOptions:`. That means
+`UIApplicationDelegate.window` and
+`UIApplicationDelegate.window.rootViewController` cannot be accessed from
+`application:didFinishLaunchingWithOptions:`.
+
+Apple is driving the adoption of the `UISceneDelegate` API since it allows apps to have multiple instances of their UIs, like multitasking on iPadOS.
+
+Previously, Flutter’s documentation would suggest that
+`application:didFinishLaunchingWithOptions:` was a good place to set up platform
+channels for users that want easy interop between the host application and
+Flutter. That is no longer a reliable place to register these platform channels
+since the Flutter engine won’t have been created yet.
+
+## Migration guide
+
+### Info.plist migration
+
+`UISceneDelegate`s must be specified in an app’s Info.plist or specified in `application:configurationForConnectingSceneSession:options:`. The Flutter tool will attempt to automatically edit the Info.plist if no `UISceneDelegate` is specified, so nothing may be required beyond running flutter run or flutter build again. Projects can be manually upgraded by adding the following to the Info.plist. `FlutterSceneDelegate` is the new class in the Flutter framework that users can use as the `UISceneDelegate`.
+
+```plist
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+ <key>UIApplicationSceneManifest</key>
+ <dict>
+  <key>UIApplicationSupportsMultipleScenes</key>
+  <false/>
+  <key>UISceneConfigurations</key>
+  <dict>
+  <key>UIWindowSceneSessionRoleApplication</key>
+    <array>
+      <dict>
+        <key>UISceneClassName</key>
+        <string>UIWindowScene</string>
+        <key>UISceneDelegateClassName</key>
+        <string>FlutterSceneDelegate</string>
+        <key>UISceneConfigurationName</key>
+        <string>flutter</string>
+        <key>UISceneStoryboardFile</key>
+        <string>Main</string>
+      </dict>
+    </array>
+   </dict>
+ </dict>
+</dict>
+```
```

Comment:
Perhaps include an image for those who do it through the GUI?

![Screenshot 2025-05-29 at 8 48 43 PM](https://github.com/user-attachments/assets/bc77642d-3af2-4000-90ca-a9258d092091)